src/main/java/com/adq/jenkins/xmljobtodsl/dsl/strategies/custom/DSLIntStrategy.java

```diff
@@ -0,0 +1,26 @@
+package com.adq.jenkins.xmljobtodsl.dsl.strategies.custom;
+
+import com.adq.jenkins.xmljobtodsl.dsl.strategies.DSLMethodStrategy;
+import com.adq.jenkins.xmljobtodsl.parsers.PropertyDescriptor;
+
+import static java.lang.Integer.parseInt;
```

Comment:
I don't see this being used, is this the latest commit?

Reply:
Forgot to remove that. It's changed now!